app/celery/scheduled_tasks.py

```diff
@@ -315,6 +315,17 @@ def beat_inbox_email_bulk():
         receipt_id_email, list_of_email_notifications = email_bulk.poll()
 
 
+if current_app.config["FF_DEBUG_STACK_TRACE"]:
```

Comment:
With an import to `env` most likely
```suggestion
if env.bool("FF_DEBUG_STACK_TRACE", False):
```